autoload/javacomplete/classpath/gradle.vim

```diff
@@ -56,8 +56,8 @@ function! s:GenerateClassPath(path, gradle) abort
   let s:temporaryGradleFile = tempname()
   let s:gradleOutput = []
   let s:gradlePath = a:path
-  let gradle = has("win32") || has("win16") ? '.\gradlew.bat' : './gradlew'
-  if !executable(gradle)
+  let gradle = javacomplete#util#IsWindows() ? '.\gradlew.bat' : './gradlew'
+  if g:JavaComplete_EnableGradle && executable('gradle')
```

Comment:
If we are here, so we already know that the `gradle` is executable (if it is not windows) (see `IfGradle` method), so we don't need to test `gradle`. And if Gradle disabled, so we shouldn't continue execution. `g:JavaComplete_EnableGradle` should be tested in `IfGradle` method. I think.